src/redux/settings.js

```diff
@@ -53,6 +61,29 @@ export const settingsLoadNetwork = () => async dispatch => {
   }
 };
 
+export const settingsLoadTestnetsEnabled = () => async dispatch => {
+  try {
+    const testnetsEnabled = await getTestnetsEnabled();
+    dispatch({
+      payload: testnetsEnabled,
+      type: SETTINGS_UPDATE_TESTNET_PREF_SUCCESS,
+    });
+  } catch (error) {
+    logger.log('Error loading testnets preference', error);
+  }
+};
+export const settingsChangeTestnetsEnabled = testnetsEnabled => async dispatch => {
+  saveTestnetsEnabled(testnetsEnabled);
+  try {
```

Comment:
not sure why you need a try/catch around this

Reply:
also we generally have a pattern of dispatching to redux before saving to localstorage